src/test/java/com/devcomanda/easymeetup/service/MeetupServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package com.devcomanda.easymeetup.service;
+
+import com.devcomanda.easymeetup.entity.Meetup;
+import com.devcomanda.easymeetup.repository.MeetupRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
```

Comment:
Your current implementation doesn't need these annotations.
You don't use spring context and another mechanism for testing your service. Your tests are good, but these annotations are redundant